CHANGELOG.md

```diff
@@ -4,16 +4,18 @@ x.x.x Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed a bug that prevented `ClientResyncMode.Discard` to discard the local changes. ([RJS-348](https://jira.mongodb.org/browse/RJS-348) and [#2664](https://github.com/realm/realm-js/issues/2664), since v3.1.0)
```

Comment:
```suggestion
* Fixed a bug that prevented `ClientResyncMode.Discard` to discard the local changes. ([#2664](https://github.com/realm/realm-js/issues/2664), since v3.1.0)
```